Tools/CISupport/ews-build/steps.py

```diff
@@ -2155,11 +2191,17 @@ def filter_out_non_reviewer_validators(candidate):
             defer.returnValue(SUCCESS)
             return
 
+        unreviewed = self.getProperty('unreviewed', False)
         for reviewer in reviewers:
             if not self.is_reviewer(reviewer):
-                rc = yield self.fail_build_due_to_invalid_status(reviewer, 'reviewer')
-                defer.returnValue(rc)
-                return
+                if unreviewed:
+                    yield self._addToLog('stdio', f'PR is marked as unreviewed. {reviewer} is not a valid reviewer.\n')
```

Comment:
Nit, but what about: `{reviewer} is not a valid reviewer, but the PR is marked as unreviewed.\n`

This makes it a bit clearer, I think.